partner/partner-management-service/src/main/resources/application-dev.properties

```diff
@@ -104,24 +88,20 @@ mosip.keycloak.admin.secret.key=admin
 # IAM
 mosip.iam.module.login_flow.name=authorization_code
 mosip.iam.module.clientID=mosip-pms-client
-mosip.iam.module.clientsecret=abc123
-mosip.iam.module.redirecturi=${mosipbox.public.url}/v1/admin/login-redirect/
+mosip.iam.module.clientsecret=JfoG3eLWLW7iSZDt
```

Comment:
can we make sure all the secrets are little tough. this is looks like some kids writing code.